.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: stable
+    hooks:
+      - id: black
+        language_version: python3.8
```

Comment:
does this commit users to python3.8 more generally?

Reply:
This follows the python3.8 standard for black, now that I think about it, it should be just `python3` rather than version specific.